test/e2e/utils/utils.go

```diff
@@ -0,0 +1,66 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package utils
+
+import (
+	"fmt"
+	"os"
+	"slices"
+	"strings"
+
+	druidv1alpha1 "github.com/gardener/etcd-druid/api/core/v1alpha1"
+
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var knownBackupProviders = []druidv1alpha1.StorageProvider{
+	"local",
+	"none",
+}
+
+func GetEnvOrError(key string) (string, error) {
```

Comment:
Nit: missing function comment (for consistency)
```suggestion
// GetEnvOrError get the environment variable by key and returns an error if it is not set.
func GetEnvOrError(key string) (string, error) {
```